crawler/crawler.conf

```diff
@@ -46,3 +46,14 @@
     # fruther data; set to 'false' or '0' to disable, enable otherwise;
     # the default value is 'false'
     terminate_fprobe = 1
+
+[ emitters ]
+
+    [[ Stdout Emitter ]]
+    arg_from_conf = 1
+    format = csv
+    
+    [[ File Emitter ]]
+    url = file://xx
```

Comment:
This is the default location for csv file emitter ? Can we put some real-path like file:///tmp/crawler-out or something ?